src/index.html

```diff
@@ -1,12 +1,386 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content">
+    <div class="container">
+      <header class="header">
+        <a href="#home">
+          <img
+            src="./images/logo.png"
+            alt="moyo logo"
+            class="header__logo"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#phones" class="nav__link">Смартфоны и телефоны</a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#computers"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Ноутбуки и компьютеры
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#gadgets" class="nav__link">Гаджеты</a>
+            </li>
+            <li class="nav__item">
+              <a href="#tablets" class="nav__link">Планшеты</a>
+            </li>
+            <li class="nav__item">
+              <a href="#photos" class="nav__link">Фото</a>
+            </li>
+            <li class="nav__item">
+              <a href="#videos" class="nav__link">Видео</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+    </div>
+
+    <div class="container">
+      <main class="main">
+        <div class="main__container">
```

Comment:
Try to avoid additional containers, you can add additional class to main tag for setting outer indents, if it's needed instead of creating div